packages/iov-crypto/src/ripemd.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { Encoding } from "@iov/encoding";
+
+import { Ripemd160 } from "./ripemd";
+import ripemdVectors from "./testdata/ripemd.json";
+
+const { toHex, fromHex } = Encoding;
+
+describe("Ripemd160", () => {
+  it("exists", () => {
+    expect(Ripemd160).toBeTruthy();
+  });
+
+  it("works for empty input", () => {
+    {
+      const hash = new Ripemd160(new Uint8Array([])).digest();
+      expect(toHex(hash)).toEqual("9c1185a5c5e9fc54612808977ee8f548b2258d31");
```

Comment:
Can we do this comparison on the binary data directly instad of the hex representation?